Makefile

```diff
@@ -0,0 +1,14 @@
+deploy:
+	docker-compose up -d --build
+
```

Comment:
we don't be deploying using docker-compose
```suggestion
```

Reply:
Removed